src/vivarium_public_health/population/base_population.py

```diff
@@ -527,3 +527,31 @@ def _assign_demography_with_age_bounds(
     )
     register_simulants(simulants[list(key_columns)])
     return simulants
+
+
+def _find_bin_start_index(value: int, sorted_reference_values: list[int]) -> int:
+    """Finds the index of the closest reference value less than or equal to the provided value.
+
+    Parameters
+    ----------
+    value
+        The value for which to find the closest reference value.
+    sorted_reference_values
+        A sorted list of reference values.
+
+    Returns
+    -------
+        The index of the closest reference value less than or equal to the provided value.
+
+    Raises
+    ------
+        ValueError
+            If the provided value is less than the minimum reference value.
+    """
+    ref_value_index = np.digitize(value, sorted_reference_values).item() - 1
+    if ref_value_index < 0:
+        raise ValueError(
```

Comment:
See my comment in the description of this PR. Basically, vivarium does a similar thing in one location and if the value falls below the reference values, it chooses the minimum instead of erroring. I'm happy to do that if we think it's better.